pages/Useful Utilities/Must-have.md

```diff
@@ -57,13 +57,9 @@ password whenever an app wants to elevate its privileges.
 Our recommendation is the KDE one. For arch, it's `polkit-kde-agent`.
 
 You can autostart it with
-`exec-once=/usr/lib/polkit-kde-authentication-agent-1`. On some distributions
-you might have to use a different path which could be
-`/ usr/libexec/polkit-kde-authentication-agent-1`,
-`/usr/libexec/kf5/polkit-kde-authentication-agent-1`, or
-`/usr/libexec/kf6/polkit-kde-authentication-agent-1`.
+`exec-once=systemctl --user start plasma-polkit-agent`.
 
-On other distributions that use a more recent version, such as Gentoo, it may be
+On other distributions that use a different init system, such as Gentoo, it may be
```

Comment:
the removed note about paths is now needed here

Reply:
Sorry for not replying to this sooner, I added back all of the other paths. Although I think the kf5 and kf6 ones are just for fedora, which has been using systemd since forever